Readme.md

```diff
@@ -1,16 +1,85 @@
-# MusicBrainz Android app
+<h1 align="center">
+  <br>
+  <a href="https://github.com/ArmynC/ArminC-AutoExec/archive/master.zip"><img src="https://github.com/metabrainz/metabrainz-logos/blob/master/logos/MusicBrainz/SVG/MusicBrainz_logo.svg" alt="MusicBrainz Android"></a>
+</h1>
 
-The official android app for MusicBrainz. MusicBrainz for Android lets users search metadata all of the the metadata available on MusicBrainz. Users can scan barcode to search albums. Registered users can view, edit and modify their collections. Users can also tag their audio files.
+<h4 align="center">MusicBrainz App</h4>
 
-# Developers
-MusicBrainz for Android was first created in 2010-11 as a part of GSoC by Jamie McDonald. The project went stagnant for a few years.
+<p align="center">
+    <a href="https://github.com/metabrainz/musicbrainz-android/commits/master">
+    <img src="https://img.shields.io/github/last-commit/metabrainz/musicbrainz-android.svg?style=flat-square&logo=github&logoColor=white"
+         alt="GitHub last commit">
+    <a href="https://github.com/metabrainz/musicbrainz-android/issues">
+    <img src="https://img.shields.io/github/issues-raw/metabrainz/musicbrainz-android.svg?style=flat-square&logo=github&logoColor=white"
+         alt="GitHub issues">
+    <a href="https://github.com/metabrainz/musicbrainz-android/pulls">
+    <img src="https://img.shields.io/github/issues-pr-raw/metabrainz/musicbrainz-android.svg?style=flat-square&logo=github&logoColor=white"
+         alt="GitHub pull requests">
+</p>
+      
+<p align="center">
+  <a href="#about">About</a> •
+  <a href="#installation">Installation</a> •
+  <a href="#downloading">Downloading</a> •
+  <a href="#developers">Developers</a> •
+  <a href="#wiki">Wiki</a> •
+  <a href="#contributing">Contributing</a> •
+  <a href="#issues">Issues</a> •
+  <a href="#support">Support</a> •
+  <a href="#license">License</a>
+</p>
+
+---
+
+## About
+
+MusicBrainz for Android is a mobile app for viewing MusicBrainz data on the Android platform. It allows you to:
+
+* View release information by scanning a barcode
+* Search for information about artists, releases, release groups,labels, recordings, instruments, and events
+* View collections
+* Tag audio files like Picard
+* Donate to the MetaBrainz Foundation via PayPal
+
+## Installation
+	    
+* First start with installing Android Studio on your pc.
```

Comment:
You can instead write this as a prerequisite to develop the app.

Reply:
Done